components/Chat/Chat.tsx

```diff
@@ -1,21 +1,12 @@
-import { useChatProvider } from "@/providers/ChatProvider";
+import { ScrollTo } from "react-scroll-to";
 import ChatInput from "./ChatInput";
 import Messages from "./Messages";
-import Suggestions from "./Suggestions";
 
 const Chat = () => {
```

Comment:
Use a ref for the Messages component container to control the scroll position directly. This approach is efficient and avoids the need for additional libraries.